packages/scripts/scripts/plugin-zip.js

```diff
@@ -47,10 +50,30 @@ if ( hasPackageProp( 'files' ) ) {
 	);
 }
 
+if ( noRootFolderArg !== undefined ) {
+	zipRootFolder = '';
+	stdout.write( 'Plugin files will be zipped without a root folder.\n\n' );
```

Comment:
Nit: I suggest better spacing as explained in https://github.com/WordPress/gutenberg/pull/61375/files#r1818530835

```suggestion
	stdout.write( '  Plugin files will be zipped without a root folder.\n\n' );
```